server/form-pages/assess/assessApplication/suitablityAssessment/suitabilityAssessment.ts

```diff
@@ -0,0 +1,148 @@
+import type { TaskListErrors, YesOrNo } from '@approved-premises/ui'
+
+import { Page } from '../../../utils/decorators'
+import { sentenceCase } from '../../../../utils/utils'
+
+import TasklistPage from '../../../tasklistPage'
+
+export const suitabilityAssessmentSections: Array<SuitabilityAssessmentSection> = [
+  {
+    id: 'riskFactors',
+    question: 'Does the application identify the risk factors that an Approved Premises (AP) placement can support?',
+    guidanceNotes: `<p>Consider:</p>
```

Comment:
I'm wondering if this HTML is better placed in Nunjucks partials, as there's a lot of it. So we could have them in 
`views/assessments/pages/partials/$ID` and then load them in that way. Then the sections could within the Page class instance itself like so:

```
export default class SuitabilityAssessment implements TasklistPage {
   ....

   sections = [
      {
          id: 'riskFactors',
          question: 'Does the application identify the risk factors that an Approved Premises (AP) placement can support?'
      },
      ...
   ]
}
```

And then used in the view like so:

```
{% for section in page.sections %}
        {{ formPageRadios({
                fieldName: section.id,
                fieldset: {
                    legend: {
                        text: section.question,
                        classes: "govuk-fieldset__legend--s"
                    }
                },
                items: [
                    {
                        value: "yes",
                        text: "Yes"
                    },
                    {
                        value: "no",
                        text: "No"
                    }
                ]
            }, fetchContext()) }}

        <details class="govuk-details" data-module="govuk-details">
            <summary class="govuk-details__summary">
                <span class="govuk-details__summary-text">
                          View guidance notes
                        </span>
            </summary>

            <div class="govuk-details__text">
                {% include './partials/' + section.id + '.njk' %}
            </div>
        </details>

        {{ formPageTextarea({
                    fieldName: section.id + "Comments",
                    label: {
                        text: "Additional comments",
                        classes: "govuk-label--s"
                    }
                }, fetchContext()) }}
{% endfor %}
```

That way we keep the page class neater without a ton of HTML, and we don't have to pass the sections variable to Nunjucks.

Whaddya reckon?

Reply:
Yeah seems smart, I'll give it a go